index.html

```diff
@@ -42,205 +59,428 @@ <h1>I'm Joseph Glad to meet you!</h1>
-            <div class="details">
-                <img src="Images/Counter.png" alt="">
-                <p>Backend Dev</p>
+            <div class="lang-desktop">
+                <h3>Languages</h3>
+                <img src="Images/arrow-down-sign-to-navigate.png" alt="">
             </div>
-            <div class="details">
-                <img src="Images/Counter.png" alt="">
-                <p>2015</p>
-            </div>
-        </div>
-        <div class="content">
-            <p>
-                A daily selection of privately personalized reads; 
-                no accounts or sign-ups required.
-            </p>
+    
+            <ul class=" basic-desktop">
+                <li class="stack-desktop"><img src="Images/Ellipse 1.png" alt="">
+                    <p>Javascript</p>
+                </li>
+                <li class="stack-desktop"><img src="Images/Ellipse 1 (1).png" alt="">
+                    <p>HTML</p>
+                </li>
+                <li class="stack-desktop"><img src="Images/Ellipse 1 (2).png" alt="">
+                    <p>CSS</p>
+                </li>
+            </ul>
+    
+            <ul class="lang-skills-desktop">
+                <li class="framework-desktop">
+                   <a href="">Frameworks</a>  <img class="image-desktop" src="Images/right-arrow.png" alt="">
+                </li>
+                <li><img class="shape-desktop" src="Images/Separator Bottom.png" alt=""></li>
+                <li class="framework-desktop">
+                   <a href="">Skills</a>  <img class="image-desktop" src="Images/right-arrow.png" alt="">
+                </li>
+                <li><img class="shape-1-desktop" src="Images/Separator Bottom.png" alt=""></li>
+            </ul>
+    
         </div>
-        
-        <ul class="link">
-            <li><a href="">Html</a></li>
-            <li><a href="">css</a></li>
-            <li><a href="">javascript</a></li>
-         </ul>
-
-         <button class="button"><a href="">See Project</a></button>
-        
+        </section>   
+    </div>
+  
+    <!-- This is the contact page -->
+    <section id="contact">
+        <h2>Contact Me</h2>
+        <p>
+            If you have an application you are interested in developing,
+            a feature that you need built or a project that needs coding.
+            I’d love to help with it
+        </p>
+        <form action="https://formspree.io/f/xgebkzlz" method="post">
+            <input name="Name" placeholder="Yeremias" type="text" maxlength="30" required>
+            <input name="Email" placeholder="natajayanj14@gmail.com" type="email" required>
+            <textarea placeholder="Write your message here" name="Message" id="" cols="30" rows="10" maxlength="500"
+                type="text" required></textarea>
+            <button type="submit" id="contact-button">Get in touch</button>
+        </form>
+        <div class="bottom"><img src="Images/Shape.png" alt=""></div>
     </section>
```

Comment:
- [ ] I also have a look at your mobile version it looks nice :clap: but I noticed that there is a `line-bar` on the bottom of your page which makes it looks ugly , kindly fix this for your mobile version to still look more beautiful :point_down:  also remove that space below :wink: 
This is how it looks like on your design 
![Screenshot from 2023-03-02 16-10-08](https://user-images.githubusercontent.com/88894525/222453061-d543f84e-1d64-449f-b234-26c92d43ccef.png)

Figma
![Screenshot from 2023-03-02 16-18-35](https://user-images.githubusercontent.com/88894525/222454200-3a29abea-d4d3-47dc-ba30-d55fa737b1ab.png)